src/styles/overrides.scss

```diff
@@ -65,7 +65,7 @@
 }
 
 .pc-navigation {
-    z-index:  101;
+    z-index:  5000;
```

Comment:
why is it 5000 and not 102 for examle ? what is z-index of pop-up ?

Reply:
Pop-ups have a z-index: 1000, and for the header I set 5000, because the header should be above all the elements on the page. In addition, an element may appear that should be above the pop-up, but under the header. It can have a z-index: 1001, 1002, etc

Reply:
It is always better to choose the smallest number to the required number, than to increase it by a random amount, as this can very quickly lead to too large numbers

Reply:
Okay, I get it. Can I use the value 1050? To have a stock of z-index values